src/thor/triplegbuilder.cc

```diff
@@ -92,6 +93,22 @@ void AssignAdmins(const AttributesController& controller,
   }
 }
 
+// Helper function to get the iso country code from an edge
+inline std::string country_code_from_edge(const graph_tile_ptr& tile,
+                                          const valhalla::baldr::DirectedEdge* de) {
+  return tile->admininfo(tile->node(de->endnode())->admin_index()).country_iso();
```

Comment:
Does this need a null-pointer check of `de` or could a reference be passed instead of the pointer?